src/main/java/seedu/address/model/country/Country.java

```diff
@@ -66,4 +71,12 @@ public String getCountryName() {
     public String getCountryCode() {
         return countryCode;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (!(obj instanceof Country)) {
+            return false;
+        }
+        return countryCode.equals(((Country) obj).countryCode);
```

Comment:
might wanna change this equals to match the 3 step thing

Reply:
Updated